xcode/Subconscious/Shared/Parsers/Subtext.swift

```diff
@@ -722,3 +724,21 @@ extension Subtext {
         return shortlinkFor(index: range.lowerBound)
     }
 }
+
+extension Subtext {
```

Comment:
Would prefer to add this extension in the SubtextTextEditor file. That way SubtextTextEditor knows about Subtext and NSTextParagraph, but Subtext doesn't have to know about NSTextParagraph.